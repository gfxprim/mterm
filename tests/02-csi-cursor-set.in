10 10
\e[2;2Htest
