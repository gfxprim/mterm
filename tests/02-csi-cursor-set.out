 ----------
|          |
| test     |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 1x5
